### ISSUE TITLE
Custom Channel Mapping Issue - Some Channels work, some don't

### ISSUE THREAD

**Issue by fortepc** [Bjornskjald/miscord#191]
_If you have a question, ask on the [Miscord Support server](https://discord.gg/DkmTvVz) instead_  
## Expected Behavior

I have two FB chats I'm trying to map to two channels in a single discord server. I expect both channels to work both ways. 

## Actual Behavior

The first chat works both ways, messages sent from Discord appear in Facebook, and messages sent in facebook appears in discord.

The second chat only works one way. Messages sent from discord do not appear in facebook. Error `This person isn\'t available right now.` Messages sent from Facebook appear in Discord

## Steps to Reproduce the Problem

  1. Set up config with 2 fb chats whitelisted, and the two channels custom mapped to two specific discord channels within the same server.
  1. Run program and send message from second discord channel
  1. See message fail to send.

Silly Logs of test sending messages from both chats from both platforms https://pastebin.com/ys97fbhS
## Specifications

  - Platform: Windows 10 ver. 1709 (x64)
  - Version (`miscord -v`): 3.0.11-win-x64
  - Node.js version (`node -v`): v8.9.4

**Comment by fortepc** [comment]
Attempted workaround fix, launching the program from a batch file instead directly launching the exe gives me the option to do this

```
@Echo off
:Start
C:\miscord.exe
echo Program terminated at %Date% %Time% with Error %ErrorLevel% >> c:\logs\program.log 
echo Press Ctrl-C if you don't want to restart automatically
ping -n 10 localhost

goto Start
```

**Comment by Bjornskjald** [comment]
```
sill login: channels   null => TextChannel {
```
That's... not what I expected to happen, after I added filtering channels by empty topic...

**Comment by Bjornskjald** [comment]
Oh, wow.
https://github.com/Bjornskjald/miscord/commit/ea9d328b0f4c963277d95b795008a28b0356dea0#diff-e071978542c958c1503d5f9036b63807
https://github.com/Bjornskjald/miscord/commit/30119509ba038fb5d5b1682bc331c78b5e807866#diff-e071978542c958c1503d5f9036b63807
I added it in 3.0.12, but didn't merge it to master/publish binaries yet...

**Comment by fortepc** [comment]
I'm still having this issue running 3.0.12 windows x64 version.

**Comment by Bjornskjald** [comment]
@fortepc just remove the topic from custom mapped channel (`1`) and everything would work as expected

**Comment by fortepc** [comment]
Still having issues. Now the Discord -> Facebook works in both channels, but the FB -> Discord messages don't work in either channel. This is the error that happens https://media.discordapp.net/attachments/431471556540104726/434117534841569281/unknown.png?width=1371&height=185

**Comment by fortepc** [comment]
Log File: https://pastebin.com/Sdk4ry4q

**Comment by Bjornskjald** [comment]
Fixed in #195.
